x/storage/keeper/permission.go

```diff
@@ -28,6 +28,16 @@ var (
 	}
 )
 
+func (k Keeper) checkIfGroupPolicy(ctx sdk.Context, groupID math.Uint) bool {
```

Comment:
can we have a better name for this method

Reply:
I directly move the `VerifyPolicy` function to the storage module and no need to check group existence after verify policy. Please review again, Thx.